articles/key-vault/key-vault-ovw-storage-keys.md

```diff
@@ -162,20 +157,19 @@ Add-AzureKeyVaultManagedStorageAccount -VaultName yourtest1 -Name msak01 -Accoun
 
 ### Regeneration
 
-Setting the regeneration period using the following commands.
+Setting the regeneration period to 3 days using the following commands.
 
 ```powershell
-$regenPeriod = [System.Timespan]::FromDays(3)
-Add-AzureKeyVaultManagedStorageAccount -VaultName yourtest1 -Name msak01 -AccountResourceId /subscriptions/subscriptionId/resourceGroups/yourresgroup1/providers/Microsoft.Storage/storageAccounts/yourtest1 -ActiveKeyName key2 -RegenerationPeriod $regenPeriod
+Add-AzureKeyVaultManagedStorageAccount -VaultName yourtest1 -Name msak01 -AccountResourceId /subscriptions/subscriptionId/resourceGroups/yourresgroup1/providers/Microsoft.Storage/storageAccounts/yourtest1 -ActiveKeyName key2 -RegenerationPeriod "03.00:00:00"
 ```
 
 ### Set SAS definitions
 
 Set the SAS definitions in Key Vault for your managed storage account.
 
 ```powershell
-Set-AzureKeyVaultManagedStorageSasDefinition -Service Blob -ResourceType Container,Service -VaultName yourtest1  -AccountName msak01 -Name blobsas1 -Protocol HttpsOnly -ValidityPeriod ([System.Timespan]::FromDays(1)) -Permission Read,List
-Set-AzureKeyVaultManagedStorageSasDefinition -Service Blob -ResourceType Container,Service,Object -VaultName yourtest1  -AccountName msak01 -Name blobsas2 -Protocol HttpsOnly -ValidityPeriod ([System.Timespan]::FromDays(1)) -Permission Read,List,Write
+Set-AzureKeyVaultManagedStorageSasDefinition -Service Blob -ResourceType Container,Service -VaultName yourtest1 -AccountName msak01 -Name blobsas1 -Protocol HttpsOnly -ValidityPeriod "01.00:00:00" -Permission Read,List
```

Comment:
@BrucePerlerMS I prefer it the original way here Bruce I think it is more programmatic.

Reply:
Agreed. I'll reject the PR.